Xamarin.Forms.Core/Layout.cs

```diff
@@ -104,8 +104,12 @@ void IPaddingElement.OnPaddingPropertyChanged(Thickness oldValue, Thickness newV
 		{
 			InvalidateLayout();
 		}
-
-		internal ObservableCollection<Element> InternalChildren { get; } = new ObservableCollection<Element>();
+		
+		/// <summary>For internal use by the Xamarin.Forms platform.</summary>
+		/// <value>To be added.</value>
+		/// <remarks>To be added.</remarks>
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
your comment is in violation with your example.

Reply:
It's just there to match Page. I would hope to spark a discussion around making it more available with a new property name (leaving the existing one obsolete).

What do you suggest?

Reply:
@StephaneDelcroix bump.
What would you suggest to fix this?

Reply:
How can I improve the comment?

Reply:
Just erase the comments per `Page`.